src/styles/blocks/product-card.css

```diff
@@ -0,0 +1,89 @@
+.product-card {
+  display: block;
+  width: 200px;
+  padding: 16px;
+  padding-top: 32px;
+
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+
+  color: #060b35;
```

Comment:
suggestion: you can add the css variables for colors to the normalize.css, add type it in this form `color: var(--name-of-css-variables)` in your css files. it will help to understand the code better